src/components/Modals/TextInputModal.vue

```diff
@@ -5,10 +5,13 @@
       <div class="textInput-wrapper">
         <input
           class="textInput-userinput"
-          maxlength="charLimit"
+          maxlength="15"
```

Comment:
the 15-character limit works for me! do we also want to require that the plan name cannot be empty?